doc/changelog.rst

```diff
@@ -1,6 +1,36 @@
 Changelog
 =========
 
+Changes in Version 4.3
+----------------------
+
+PyMongo 4.3 brings a number of improvements including:
+
+- Improved support for documents with BSON datetimes outside of the range
```

Comment:
"Improved support for documents with" -> "Added support for decoding BSON datetimes outside of the range supported by Python's datetime.datetime."